tests/test_connection.py

```diff
@@ -791,6 +791,21 @@ def test_channel_persistence(node_factory, bitcoind, executor):
     l1.daemon.wait_for_log(' to ONCHAIN')
 
 
+def test_private_channel(node_factory):
+    l1, l2 = node_factory.line_graph(2, fundchannel=True, announce=True, privatechannels=True)
+    l3, l4 = node_factory.line_graph(2, fundchannel=True, announce=True, privatechannels=False)
```

Comment:
fundchannel defaults True already AFAICT?  We should probably rename the 'announce' parameter to 'wait_for_announce'?